pkgs/tools/filesystems/apfsprogs/default.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation {
+  pname = "apfsprogs-unstable";
+  version = "2021.05.07";
```

I would like to suggest to move `unstable` to `version`. No need to touch `pname` if the package is updated.

```suggestion
  pname = "apfsprogs";
  version = "unstable-2021-05-07";
```

see https://nixos.org/manual/nixpkgs/stable/#sec-package-naming

---

I also read this and interpreted "Also append "unstable" to the name" as appending to `pname`. But if you say that prepending "unstable" to the version instead is correct, then I'll do that.